src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMBooster.scala

```diff
@@ -160,7 +237,42 @@ class LightGBMBooster(val model: String) extends Serializable {
     lightgbmlib.intp_value(numClassesOut)
   }
 
-  private def predToArray(classification: Boolean, scoredDataOutPtr: SWIGTYPE_p_double, kind: Int): Array[Double] = {
+  /**
+    * Retrieve the number of models per a iteration from LightGBM Booster
```

Comment:
(minor doc nit):
Retrieve the number of models per each iteration from LightGBM Booster
@return The number of models per iteration.

Reply:
It is fixed.